backend/src/openarchiefbeheer/destruction/managers.py

```diff
@@ -4,6 +4,17 @@
 
 
 class DestructionListQuerySet(QuerySet):
+    def permitted_for_user(self, user: User):
+        """
+        Returns a queryset filtered to items visible to the given user.
+        Visibility is granted if the user has the 'accounts.can_start_destruction'
+        permission or is assigned to the list.
+        """
+        if user.has_perm("accounts.can_start_destruction"):
+            return self
+
+        return self.filter(assignees__user__in=[user.pk])
```

Comment:
From the acceptance criteria I see:

_De reviewer/archivaris/medebeoordelaar moet alleen de lijsten zien in de fasen waarin hij/zij wat met de zaken moet doen_ 

But with this filter, you are not getting the lists to which a user is currently assigned, but the lists in which they will be/were assigned at some point :thinking: 

So, maybe this should be:
```python
return self.filter(assignee=user)
```

Reply:
This would mean that the landing page becomes largely useless (as would be the story for filtering). I think we should interpreted as "is somehow involved in the process". Which IMO makes more sense since a user probably needs to know what happend to a list afterwards.

Reply:
As discussed: I asked for clarification in https://dimpact.atlassian.net/jira/software/c/projects/PZ/boards/258?selectedIssue=PZ-4613. 

For now we leave it as is but may come back to this later.